index.js

```diff
@@ -1,7 +1,160 @@
+        <h2 class="work-title">${work.title}</h2>
+        <div class="works-details">
+          <h4 class="work-canopy">${work.platform}</h4>
+          <p class="profession">
+            <span class="prof-span">
+              <img src="./images/Counter.png" alt="counter" /></span
+            >${work.stack}
+          </p>
+          <p class="profession">
+            <span class="prof-span">
+              <img src="./images/Counter.png" alt="counter" /></span>${work.price}
+          </p>
+        </div>
+        <p class="work-info">  ${work.description} </p>
+        <div class="tech-highligt">
+          <p>HTML</p>
+          <p>CSS</p>
+          <p>Javascript</p>
+        </div>
+        <button class="tech-project-btn" type="submit" id=${work.id}>See project</button>
+      </div>
+    </article>`);
+  displayWork = displayWork.join('');
+  worksec.innerHTML = displayWork;
+}
+generateWork();
+
+const seeProject = document.querySelectorAll('.tech-project-btn');
+seeProject.forEach((button) => {
+  button.addEventListener('click', (e) => {
+    if (e.target.classList.contains('tech-project-btn')) {
+      popContainer.style.display = 'block';
+      const id = parseFloat(e.target.id);
+      const search = details.find((x) => x.id === id);
+
+      if (search) {
+        popContainer.innerHTML = `<div class="pop-up">
+        <div class="popup-info">
+        <i class="fa-solid fa-xmark pop-close"></i>
+          <h5 class="popup-h5">${search.title}</h5>
+          
+          <div class="pop-stack">
+             <div class="pop-stackname">
+              <p class="platform">${search.platform}</p>
+              <p class="pop-counter">  <span class="prof-span">
+              <img src="./images/Counter.png" alt="counter" /></span
+            >${search.stack}</p>
+              <p  class="pop-counter"> <span class="prof-span">
+              <img src="./images/Counter.png" alt="counter" /></span
+            >${search.price}</p>
+             </div>
+            
+            <div class="popup-img">
+              <img src=${search.img} < alt=${search.title} class="popup-image">
+            </div>
+            <div class="popup-description">
+             <div class="popup-p">
+            <p class="popup-pone">${search.longdescription}</p>
+           
+             </div>
+             <div >
+             <div class="popup-techstack">
+             ${search.technology.map((x) => `<p class="popup-pone">${x}</p>`).join('')}
+             
+             </div>
+            <div class="popup-btn">
+            <a href=${search.livelink} class="see-live"
+            ><img src="./images/Enabled1.png" alt="see live" class= "pop-enabled"
+          /></a>
+            <a href=${search.livelink} class="see-live"
+             ><img src="./images/Enabled.png" class= "pop-enabled" alt="see live"
+          /></a>
+                
+                </div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Good work adding the pop up dynamically as is required. Please make sure the popup is responsive, it fits mobile and desktop sizes properly. This is a project requirement. Currently both desktop & mobile pop ups do not fit properly on the screen. 🥇 
![scroll](https://github.com/blessing-michael/portfolio/assets/99656687/3b1ff953-c064-4eff-93fb-1964b5c66cb2)

Reply:
Thanks for the review but [wayungi](https://github.com/wayungi) please can you explain further as i do not understand the fit properly on screen. It looks fine on the screen and I don't seem to see the issue you are referring to.  if it's possible add a picture to aid my visual understanding.Thank you